invisible_cities/io/dst_io.py

```diff
@@ -75,6 +75,7 @@ def store_pandas_as_tables(h5out : tb.file.File, df : pd.DataFrame, group_name :
         warnings.warn(f'dataframe is empty', UserWarning)
     else:
         _can_cast(arr, data_types)
-        arr = arr.astype(data_types)
+        columns = list(data_types.names)
+        arr = arr[columns].astype(data_types)
```

Comment:
where is the sorting?

Reply:
It is not sorted, is just assured that the columns of record array have the same order as the ones in table

Reply:
OK. Got slightly confused by the commit message.